test/parallel/test-worker-unsupported-path.js

```diff
@@ -13,6 +13,7 @@ const { Worker } = require('worker_threads');
   assert.throws(() => { new Worker('/b'); }, expectedErr);
   assert.throws(() => { new Worker('/c.wasm'); }, expectedErr);
   assert.throws(() => { new Worker('/d.txt'); }, expectedErr);
+  assert.throws(() => { new Worker(new URL('file:///e.wasm')); }, expectedErr);
```

Comment:
@addaleax I don't have a Windows setup to test this, would this change be enough to fix the CI error?
```suggestion
  assert.throws(() => { new Worker(new URL('file:///C:/e.wasm')); }, expectedErr);
```

Reply:
I’m not usually using Windows myself but I can check sometime later today or tomorrow

Reply:
Yeah, looks like that works :)

Reply:
Regardless of platform, file URLs are required to be absolute paths.

Reply:
@jasnell `/e.wasm` can be parsed as an absolute path on UNIX-platform, but not on Windows. `/C:/e.wasm` on the other hand can be parsed as an absolute path on all supported platforms.